Translation Completed for `library/cmath`

whiteSHADOW1234 opened the pull request:
## Summary
This pull request marks the completion of the `cmath` Python library section translation, transitioning it from English to Traditional Chinese (TW). The changes include crucial improvements, transparent reasoning for translation choices, and strict adherence to [established translation rules (翻譯守則)](https://github.com/python/python-docs-zh-tw?tab=readme-ov-file#%E7%BF%BB%E8%AD%AF%E5%AE%88%E5%89%87).

## Details
In addressing words not listed in the [attached Glossary](https://github.com/python/python-docs-zh-tw?tab=readme-ov-file#%E8%A1%93%E8%AA%9E%E8%A1%A8-glossary), the following words have been translated, providing links justifying the choice:
  - **"branch cut"** - 分枝切割 [link](https://sheethub.com/data.gov.tw/%E5%9C%8B%E5%AE%B6%E6%95%99%E8%82%B2%E7%A0%94%E7%A9%B6%E9%99%A2-%E6%95%B8%E5%AD%B8%E5%AD%B8%E8%A1%93%E5%90%8D%E8%A9%9E/uri/56643469)
  - **"real axis"** - 實數軸 [link](https://sheethub.com/data.gov.tw/%E5%9C%8B%E5%AE%B6%E6%95%99%E8%82%B2%E7%A0%94%E7%A9%B6%E9%99%A2-%E6%95%B8%E5%AD%B8%E5%AD%B8%E8%A1%93%E5%90%8D%E8%A9%9E/uri/56654545)
  - **"imaginary axis"** - 虛軸 [link](https://terms.naer.edu.tw/detail/60b5db04285ede92cd67046b114b60f7/)
  - **"modulus"** - 絕對值 [link](https://terms.naer.edu.tw/detail/7ea23ff624ed5735baf24637b983d30e/?startswith=zh&seq=6)
      - Rationale: **modulus** is translated as **絕對值** instead of **模數** due to the original document's mention of `The modulus (absolute value) ...` in the 99th line. In mathematics, **模數** is often associated with [`mod`](https://zh.wikipedia.org/zh-tw/%E6%A8%A1%E9%99%A4).
  - **"as a"** - 作為一個 [link](https://dictionary.cambridge.org/zht/%E8%A9%9E%E5%85%B8/%E8%8B%B1%E8%AA%9E-%E6%BC%A2%E8%AA%9E-%E7%B9%81%E9%AB%94/as)
  - **"Rectangular coordinates"** - 直角坐標 [link](https://terms.naer.edu.tw/detail/77a97430f270870cb1faa2f70341601d/?seq=4)
  - **"tolerance"** - 容許偏差 [link](https://terms.naer.edu.tw/detail/d9ef649d2dc62d3ed5d2564082c2fa29/?startswith=zh)


Closes #485

whiteSHADOW1234 committed:
Translate cmath

whiteSHADOW1234 committed:
fix typo

mattwang44 commented:
@cschan1828 would you please review this one?

cschan1828 commented:
> @cschan1828 would you please review this one?

Sure. Will do.

cschan1828 commented on library/cmath.po:
```diff
@@ -275,48 +324,51 @@ msgid ""
 "A floating-point \"not a number\" (NaN) value.  Equivalent to "
 "``float('nan')``."
-msgstr ""
+msgstr "浮點\\\"非數字\\\" (NaN) 值。相當於 ``float('nan')`` 。"
```
```suggestion
msgstr "浮點「非數字」 (NaN) 值。相當於 ``float('nan')``。"
```

原文應該是要強調，改用中文的引號較適當。

cschan1828 commented on library/cmath.po:
```diff
@@ -275,48 +324,51 @@ msgid ""
 "Complex number with zero real part and NaN imaginary part. Equivalent to "
 "``complex(0.0, float('nan'))``."
-msgstr ""
+msgstr "零為實部和 NaN 為虛部的複數。相當於 ``complex(0.0, float('nan'))`` 。"
```
```suggestion
msgstr "實部為零和虛部為 NaN 的複數。相當於 ``complex(0.0, float('nan'))``。"
```

cschan1828 reviewed (commented):
LGTM.

nit: 若 `` 符號及 * 符號遇到句尾（逗號或句號）則不須留空格。

whiteSHADOW1234 committed:
Merge branch '3.12' into cmath

whiteSHADOW1234 committed:
update cmath

whiteSHADOW1234 committed:
Update cmath

whiteSHADOW1234 committed:
fix typo

whiteSHADOW1234 commented:
I've made some updates to the translated document in this pull request:

1. I've also revised the 358th line from ```零為實部和正無窮為虛部的複數。相當於 ``complex(0.0, float('inf'))`` 。``` to ```實部為零和虛部為 NaN 的複數。相當於 ``complex(0.0, float('nan'))``。```

2. I've updated the 364th line from ```浮點\\\"非數字\\\" (NaN) 值。相當於 ``float('nan')`` 。``` to ```浮點「非數字」 (NaN) 值。相當於 ``float('nan')``。```

3. The 370th line has been updated from ```零為實部和 NaN 為虛部的複數。相當於 ``complex(0.0, float('nan'))`` 。``` to ```實部為零和虛部為 NaN 的複數。相當於 ``complex(0.0, float('nan'))``。```

4. Additionally, I've accepted user cschan1828's advice and removed all the blanks between special symbols ( `` and * ) and punctuation marks (， and 。).

Please review the changes, and I'm open to any further suggestions or improvements.

Thanks!

cschan1828 reviewed (approved):
LGTM. Approval.
